### ISSUE TITLE
`cache-control` not properly parsed

### ISSUE THREAD

**Issue by erickok** [hurshi/dio-http-cache#24]
When relying on response headers for max cache time, i.e. no manually set max age, the `cache-control` header is not properly parsed, causing the whole caching to fail.

To reproduce, set up a dio get with `options: buildServiceCacheOptions()`. The request will never be served from cache.

I tried to debug the library and found that the `cache-control` header value is noty properly parsed in[`_tryParseHead`](https://github.com/hurshi/dio-http-cache/blob/master/lib/src/manager_dio.dart#L143). Currently this reads:
```dart
parameters = HeaderValue.parse(cacheControl,
                parameterSeparator: ",", valueSeparator: "=")
```
But the `HeaderValue.parse` method expects the header name to be present in the given String, so it should be:
```dart
parameters = HeaderValue.parse(HttpHeaders.cacheControlHeader + ": " + cacheControl,
                parameterSeparator: ",", valueSeparator: "=")
```

**Comment by danmork** [comment]
We hit this issue too. Any plans to merge the PR? ( @erickok, thanks for providing it. )

**Comment by hurshi** [comment]
Hello, thanks for your feedback.
I am very sorry for not replying for a long time.
And I released the new version(0.2.6) according to your opinion.
